content/blog/2020-07-01-june-release/contents.lr

```diff
@@ -0,0 +1,100 @@
+title: InvenioRDM June Release
+---
+author: Lisa O'Keefe, Pablo Panero
+---
+body:
+
+We are happy to announce InvenioRDM Alpha 8 (June release). Thank you to our team members for their efforts on this release.
+
+### What's new?
+
+The June release is a major backend development release. We developed three new modules that lay the foundation for the new core data flow of Invenio(RDM). Our work goes far to provide a better developer experience. You will be able to work on and customize Invenio(RDM) in an easier and cleaner way.
+
+The primary focus for the release was:
+
+- Continue the migration from Bootstrap to SemanticUI
+- Bug fixes and improvements on the frontend (mainly on deposit)
+- Develop new core modules for Invenio(RDM) backend
+
+#### UI Customization
+
+You are able to customize your templates again. The JSX customizations are not functional yet though, which means the search page results are not customizable.
+
+#### Visual appearance
+
+We have migrated the file previewer, the OAuth login, and the settings pages to SemanticUI.
+
+You might still notice some issues related to visual appearance (e.g. the name previewed file is not in a panel as before). We will be working on making all this perfect in the next sprint on SemanticUI (end of August). In the meantime, it is, at least, better.
+
+![](record_page.jpeg)
+
+#### Creators, Contributors and Affiliations Identifiers
+
+The Deposit page now accepts [ORCID iDs](https://orcid.org/) for people (creators and contributors) and [ROR identifiers](https://ror.org/) for organizations (creators, contributors and affiliations). An icon with a link to the respective entity on the record landing page reflects this connection.
+
+Contributors are now optional on the deposit page as well.
+
+#### Python versions
+
+There are now Invenio images for Python 3.7 and Python 3.8! This means that you can use Python 3.6, 3.7 and 3.8 for development and for containerize.
```

Comment:
for containerize?

Reply:
Is the concept we have (`--containerize`), meaning you are running the instance in containers.

Reply:
Oh! Now I see. I guess while reading it sounded weird. Maybe if they were written like `development` and `containarize` or even including the `--` (`--development`...I would have guessed it :)

Reply:
Yeah that's true, and even if there is a new user that is not familiar to the `invenio-cli` that might sound weird... How could we rephrase it?

```suggestion
There are now Invenio images for Python 3.7 and Python 3.8! This means that you can use Python 3.6, 3.7 and 3.8 for local development and the full containerized option.
```
?

Reply:
LGTM, thanks